.github/workflows/fpga.yml

```diff
@@ -64,6 +64,85 @@ jobs:
             RTL_VERSION="latest"
           fi
           echo "rtl_cache_key=$(git rev-parse HEAD:hw/fpga/src)-$(git hash-object hw/fpga/fpga_configuration.tcl)-$(cd hw/${RTL_VERSION}/rtl && git rev-parse HEAD)-${{ inputs.fpga-itrng }}-${{ env.CACHE_BUSTER }}" >> $GITHUB_OUTPUT
+          echo "kmod_cache_key=fpga-kernel-modules-$(git rev-parse HEAD:hw/fpga/io_module)-$(git rev-parse HEAD:hw/fpga/rom_backdoor)-${{ env.CACHE_BUSTER }}" >> $GITHUB_OUTPUT
+
+      - name: Restore kernel modules from cache
```

Comment:
Do the modules take a long time to build? The 1.x ones were pretty quick. I'm wondering if it's worth the hassle of caching them.

Reply:
It's pretty fast (~4 mins) but we need to upload the artifact to pass it to the FPGA anyways. I'll follow up with you to get more details